test/e2e/performance/perf_test.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2024 The KubeStellar Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package perf
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/onsi/ginkgo/v2"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	ksapi "github.com/kubestellar/kubestellar/api/control/v1alpha1"
+	"github.com/kubestellar/kubestellar/test/util"
+)
+
+const (
+	ns = "nginx"
+)
+
+var _ = ginkgo.Describe("performance tests", func() {
+	ginkgo.BeforeEach(func(ctx context.Context) {
+		util.CleanupWDS(ctx, wdsClient, ksWdsClient, ns)
+	})
+
+	ctx := context.Background()
+
+	ginkgo.Context("scalability", func() {
+		ginkgo.It("create a bindingpolicy and a single deployment", func() {
```

Comment:
FYI, the func here could take a `context.Context` parameter. See https://github.com/onsi/ginkgo/blob/v2.13.0/core_dsl.go#L539 .

Reply:
Done